frontend/src/theme/vars/colors.scss

```diff
@@ -8,7 +8,7 @@ $color-secondary-light-2: #f4d8ca;
 
 $color-grey-1: #4a4747;
 $color-grey-2: #5c5959;
-$color-grey-3: #6f6b6b;
+$color-grey-3: #000000;
```

Comment:
Please, don't change the colors in this file. There is a color called `$black`, you can use it